templates/aws.go

```diff
@@ -0,0 +1,44 @@
+package templates
```

Comment:
is it possible to avoid using the template and just marshal the object to an ini file like in openstack https://github.com/rancher/rke/blob/master/cloudprovider/openstack/openstack.go#L40-L51